dist/adguard.json

```diff
@@ -48,6 +48,12 @@
             "url": "https://crashlytics.com/",
             "companyId": null
         },
+        "showrss": {
+            "name": "showRSS",
+            "categoryId": 6,
```

Comment:
It does not track users data, just creates feeds with torrent magnet links (if I understand correctly). Maybe better `12`

Reply:
Yeah, it's a tracker for TV shows, not a tracker of user data.

You select a tv show, add it to your list, then export the rss feed into a third party app such as Catch for macOS to automate tv show downloads.

Reply:
hmmm, then I think it's more of a miscellaneous category - `8`